mutation_partition.cc

```diff
@@ -250,15 +250,33 @@ void mutation_partition::ensure_last_dummy(const schema& s) {
 }
 
 void mutation_partition::apply(const schema& s, const mutation_partition& p, const schema& p_schema) {
-    apply_weak(s, p, p_schema);
+    mutation_application_stats app_stats;
```

Comment:
You should get this from the caller, or you can lose some row creations. These should always be propagated from the top-most scope that either has a place to store them or knows it can throw them away (like tests).

Reply:
Now, the `mutable_application_stats` argument is required in `apply`/`apply_*` methods. I converted all invocations which previously did not supply the argument to provide a throw-away stats argument. If you would like me to, I can try propagating the stats argument higher up the call hierarchy.